packages/domain-picker/src/hooks/use-domain-suggestions.ts

```diff
@@ -61,8 +61,17 @@ export function useDomainSuggestions(
 
 			const errorMessage = getDomainErrorMessage();
 
-			return { allDomainSuggestions, state, errorMessage, retryRequest };
+			return { allDomainSuggestions, state, errorMessage };
 		},
 		[ domainSearch, domainCategory, quantity, locale, extraOptions ]
 	);
+
+	if ( ! domainSuggestions ) {
+		return;
+	}
```

Comment:
Needed to handle the case when `domainSearch` is empty or of insufficient length - then the `useSelect()` call will return simply `undefined`.